recipes/openssl/all/conanfile.py

```diff
@@ -141,6 +141,11 @@ def source(self):
     def configure(self):
         del self.settings.compiler.libcxx
         del self.settings.compiler.cppstd
+        if self._full_version >= "1.1.0":
```

Comment:
this is more correct, as "2.x" versions are coming soon

Reply:
The next version will be 3.0 and then they are following (almost) semver. So no more letters in their version numbers.

But I guess it will be much easier to rename `all` to `1-x` at that point and create a `3-x` directory